doc/documentation_workflow/developer_documentation_workflow.rst

```diff
@@ -75,5 +75,5 @@ index via the command line
 
     The ``helpdesk()`` command does not work if you have compiled
     NEST with MPI support, but you have to enter the address of the helpdesk
-    (``file:///</path/to/nest_install_dir>/share/doc/nest/index.html``) manually into the browser.
-    Replace ``/path/to/nest_install_dir`` with the path under which NEST is installed.
\ No newline at end of file
+    (``file:///<nest_install_dir>/share/doc/nest/index.html``) manually into the browser.
```

Comment:
Should it be ` ``file://<nest_install_dir>/share/doc/nest/index.html`` `? To make it consistent with the change below (in doc/getting_help.rst)

Reply:
Wow, good catch! :clap: